data/human/culture conversations.txt

```diff
@@ -615,3 +615,68 @@ mission "Sol-Bound Shuttle"
 			label end
 			`	You say your goodbyes as you board your ships. Although you are both captains, the trajectory of your lives have taken very different courses.`
 				decline
+
+mission "Burthen Triathlon"
+	minor
+	source
+		planet "Burthen"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`Burthen's planetary spaceport is not known to be well-trafficked because of the planet's high gravity, but today it seems to have a lot of visitors. Many of the usually empty landing pads are now occupied by shuttlecraft, and as you step out onto the spaceport's rather untarnished apron, you notice there's a sizable crowd at the edge of the spaceport, some of them obviously from offworld judging by their height. There's even a few media crews standing by, although it looks like a couple of their camera drones are struggling in Burthen's gravity.`
+			choice
+				`	(See what's going on.)`
+				`	(Avoid the commotion and tend to your own business.)`
+					decline
+			`	Every step takes significantly more effort than you're used to, but you eventually reach the crowd. From the signs that have been put up on the way it looks like the 'Burthen Triathlon' is taking place today, and the spaceport has been designated the finishing point. A couple of especially sturdy-looking marquees have been set up; the smaller one appears to be the administration tent, currently only accessible to event staff, while the other one is far larger and is catering food and drinks for the crowd nearby.`
+			choice
+				`	(Walk over to the catering tent.)`
+				`	(Stay outside.)`
+					goto outside
+				`	(Return to your ship.)`
+					decline
+
+			`	Thankfully the walk to the catering tent is far shorter than the walk from your ship, but nevertheless you're exhausted by the time you drag your feet inside. A lot of effort has been put into this tent: the metal chairs are all sturdy to allow offworlders not used to the gravity to pull themselves up, but cushioned as to not make it uncomfortable to sit in; a small firepit has been built in the middle of the tent, along with a couple fire extinguishers placed nearby; and the whole place is brightly lit in contrast to Burthen's typically chilly weather. Quite a few people, both Burthensiders and offworlders, are mingling about here.`
```

Comment:
```suggestion
			`	Thankfully the walk to the catering tent is far shorter than the walk from your ship, but nevertheless you're exhausted by the time you drag your feet inside. A lot of effort has been put into this tent: the metal chairs are all sturdy to allow offworlders not used to the gravity to pull themselves up, but cushioned as to not make it uncomfortable to sit in. A small firepit has been built in the middle of the tent, along with a couple fire extinguishers placed nearby, and the whole place is brightly lit in contrast to Burthen's typically chilly weather. Quite a few people, both Burthensiders and offworlders, are mingling about here.`
```

You could probably keep the second semicolon, but you can't keep both.

Reply:
I'll commit it because it does read a bit better, but I'm pretty sure semicolons can be used this way.

~~Last time I got burned for misreading and referencing a website on grammar.~~ https://www.merriam-webster.com/grammar/a-guide-to-using-semicolons

Reply:
Well, you _can_ use semicolons that way, but no one wants to read a sentence with two of them in it, especially since there's only a couple hundred in the data files in total.

Reply:
but I love semicolons!